cardano-explorer-node/src/Explorer/Node.hs

```diff
@@ -308,13 +323,40 @@ localInitiatorNetworkApplication Proxy trce pInfoConfig =
           ChainSyncWithBlocksPtcl -> \channel -> do
             liftIO $ logInfo trce "Starting chainSyncClient"
             latestPoints <- liftIO getLatestPoints
+            currentTip <- liftIO getCurrentTip
             liftIO $ logDbState trce
-            runPeer
+            actionQueue <- newSingleTypeQueueIO
+            (metrics, server) <- runRegistryT $ do
+              metrics <- makeMetrics
+              registry <- RegistryT ask
+              server <- liftIO $ async $ runReaderT (unRegistryT $ serveHttpTextMetricsT 8080 []) registry
+              pure (metrics, server)
+            dbThread <- async $ startDbThread trce actionQueue metrics
+            wut <- runPipelinedPeer
```

Comment:
what is the proper way to return `Void`? the constructor doesnt seem to be exported

Reply:
Ah missed that.

Two best way to do that are:
```
    void runPipelinedPeer
```
or
```   _ <- runPipelinedPeer
```